boards/nxp/s32k5xxcvb/doc/index.rst

```diff
@@ -0,0 +1,197 @@
++-------------------+--------------------+
+| GPIO48 - GPIO63   | PD0 - PD15         |
++-------------------+--------------------+
+| GPIO64 - GPIO78   | PE0 - PE14         |
++-------------------+--------------------+
+| GPIO80 - GPIO95   | PF0 - PF15         |
++-------------------+--------------------+
+| GPIO96 - GPIO111  | PG0 - PG15         |
++-------------------+--------------------+
+| GPIO112 - GPIO127 | PH0 - PH15         |
++-------------------+--------------------+
+| GPIO128 - GPIO138 | PI0 - PI10         |
++-------------------+--------------------+
+| GPIO160 - GPIO175 | PK0 - PK15         |
++-------------------+--------------------+
+| GPIO176 - GPIO191 | PL0 - PL15         |
++-------------------+--------------------+
+| GPIO192 - GPIO207 | PM0 - PM15         |
++-------------------+--------------------+
+| GPIO208 - GPIO223 | PN0 - PN15         |
++-------------------+--------------------+
+| GPIO224 - GPIO239 | PO0 - PO15         |
++-------------------+--------------------+
+| GPIO240 - GPIO254 | PP0 - PP14         |
++-------------------+--------------------+
+| GPIO256 - GPIO263 | PQ0 - PQ7          |
++-------------------+--------------------+
+| GPIO304 - GPIO319 | PT0 - PT15         |
++-------------------+--------------------+
+| GPIO320 - GPIO335 | PU0 - PU15         |
++-------------------+--------------------+
+| GPIO336 - GPIO351 | PV0 - PV15         |
++-------------------+--------------------+
+| GPIO352 - GPIO363 | PW0 - PW11         |
++-------------------+--------------------+
+
+LEDs
+----
+
+The board has three user RGB LEDs:
+
+=======================  =====  ====
+Devicetree node          Color  Pin
+=======================  =====  ====
+led0 / user_led_red      Red    PP1
+led1 / user_led_green    Green  PO2
+led2 / user_led_blue     Blue   PN10
+=======================  =====  ====
+
+The user can control the LEDs in any way. An output of ``1`` illuminates the LED.
+
+Buttons
+-------
+
+The board has two user buttons:
+
+=======================  =======  ===
+Devicetree node          Label    Pin
+=======================  =======  ===
+sw0 / user_button_0      USERSW0  PF3
+sw1 / user_button_1      USERSW1  PF9
+=======================  =======  ===
+
+System Clock
+============
+
+- The Arm Cortex-M7: 200 MHz.
+- The Arm Cortex-R52: 800 MHz.
+
+Set-up the Board
+================
+
+Connect the external debugger probe to the board's JTAG Cortex ETM connector (``J17``)
+and to the host computer via USB or Ethernet, as supported by the probe.
+
+NXP S32K5XX-MB Shield
+=====================
+
+This Zephyr shield, :ref:`nxp_s32k5xx_mb` expands the board’s available I/O connectivity
+
+The shield provides access to the following feature:
+- Serial UART
+
+Programming and Debugging
+*************************
+
+.. zephyr:board-supported-runners::
+
+Applications for the ``s32k5xxcvb`` board can be built in the usual way as
+documented in :ref:`build_an_application`.
```

Comment:
it looks like the board doc is added in the wrong commit

Reply:
I've updated